actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -11,6 +11,7 @@ class UnsafeRedirectError < StandardError; end
 
     included do
       mattr_accessor :raise_on_open_redirects, default: false
+      mattr_accessor :redirect_hosts_allowed, default: []
```

Comment:
As this accessor is a collection of hosts, what do you think about renaming the accessor?
```suggestion
      mattr_accessor :allowed_redirect_hosts, default: []
```

Reply:
Yes, it's better! I changed that 👍🏻